server/src/main/java/org/elasticsearch/index/IndexMode.java

```diff
@@ -324,48 +336,33 @@ protected static String tsdbMode() {
         return "[" + IndexSettings.MODE.getKey() + "=time_series]";
     }
 
-    public static final CompressedXContent DEFAULT_TIME_SERIES_TIMESTAMP_MAPPING;
+    private static CompressedXContent createMapping(boolean includeHostName) throws IOException {
```

Comment:
nit: Could we add `default` to the method name? I think `createMapping` is quite generic and might cause confusion.